recipes/CRISPRStudio/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version ="1" %}
+
+package:
+ name: crispr_studio
+ version: {{ version }}
+
+source:
+ - url: https://github.com/moineaulab/CRISPRStudio/archive/v1.tar.gz
+   sha256: 23cb08bdb851c473ba6275e40e9fdd9aedcb6590d1f53cc04cae776fb39e2fa8 
+ - url: http://faculty.virginia.edu/wrpearson/fasta/CURRENT/fasta-36.3.8g-linux64.tar.gz
+   sha256: 8aafc7cfff7286282439bafba327aa432ae4bba1917f2c96b0dc3e4433af4699
+   folder: fasta-36.3.8g
+ 
+build:
+ number: 0
+
+requirements:
+ build:
+  - {{ compiler('c') }} 
```

Comment:
You're not compiling anything, you can omit this

Reply:
Are you sure? As I'm downloading fasta36 and if I remember correctly I added this line because the files in fasta36 weren't getting linked

Reply:
That's actually another issue, that fasta36 is already a package, so just list it as a run time dependency.

Reply:
the CRISPR studio script has the path to look for fasta36 hardcoded to a certain location. Which is why I did it this way.

Reply:
Can you patch that?

Reply:
Made a change that seems to work. It just introduced a `ClobberWarning` that I can't seem to get rid of.